src/core/org/luaj/vm2/LuaDouble.java

```diff
@@ -21,6 +21,8 @@
 ******************************************************************************/
 package org.luaj.vm2;
 
+import jdk.nashorn.internal.objects.Global;
```

Comment:
This is not needed and breaks compilation.

Reply:
Fixed, sorry!